posts/2023-06-22-mfdataset-preprocess.ipynb

```diff
@@ -0,0 +1,8180 @@
Wi32C8np2MzKEHUh77Fmk/mnzpCO3ULhhgzDMqS4ArWLxSUjGMDVZ56oZByJVIZkf08z5iy+440hI3bQi9R6ICovXRkyu8nczhHfZZMlyILYUcT9a2zZsgUAMG7cOKxbtw7Hjh1DeXk53nzzTZSWluLdd9/FiBEjEjXOtMGaTZagmKEYqfXxWlAChmUocmEbliGFuZTSAdTN72ugZchXWLPJVFqG3AdQ52UFUZDtPsNNxwigVrCwm6e9l3VbEuX+6GprUZOp4FrNCAaM8dY3hV3GDLV+L46Xk3t3Qpe8TPTpkgdAd5Mlph2H166pZI8ZivsuMHz4cJx22mmYP38+Zs2ahWXLliVyXGmLxTLURsyQmx2aft6gFlW+nFpfgIhCpLLjtE7LAGpn8UzmAOokvCZTjkRVoJat5WNedEs75SiNdTxrQHe8//k3GNG3i+tzWbNHXZ9OGtUVqHWCAQ1ZwYBUF/e2yMkM4tv6JhxvDLmKGXJbZwgAnrtmNBpCYfyn+jiA5nYcegA1s8l8Rdy/xrp16zB8+HD8+Mc/Rs+ePTFnzhysWbMmkWNLSyyWoRbZZGo074DZMqTHDMV5IzLfIEJhgVAoATFDktkr9lYhAc37EvXEahlS4iZr/kllFyjznCpR6CIDgO+ecSLeW3Q+Bvfs5PpcAUVuMreYxax6c1Fkmg+qrAlGf7KmkDvLkMs6Q0DEWpKXlWH8luY6Q6rdZF7f6tKmztDYsWOxZMkS7N+/H0888QT27t2L8847D+Xl5fj5z3+Offv2JXKcaYO1AnXrFhI3c02fqJqmoaK0AFkZAZzSu6ido6zHApEg6oRkk0lm0djfSxeZP9BT6zODmiV+SPp8zTvqHMkgZfN1Vdopp413yqFKAU+URcbxOBKolJkb96qSW7ap8KKqCtSydYZ0zLGWjYosYX5r1Jrhk/kqi+NfODc3F1dddRXeeust7Ny5E1dccQV++9vf4sQTT8SFF16YiDGmFZY6Q63EDAUD8XWYb+8zggGgb7d8vL/ofPz8klPiOtZ8MwyFExMzZM2icXCcz+pukAh6unNRbqaSBe/6Cf0xfXgfjD2pu9TxZotSaSe1liGVWOoMeWoZin62akurCkuhnWh/spCroovW3mLuvrd+LwoLhV3rfZZar5k8DclY381V5GB5eTl+/OMfo6ysDHfddRf++te/qhpX2hJPNpnbiWZ0rW8+T76DAFKzghESIiHZZGYdkJah5Ce3uQKyinghADh/SCnOH1Iqfbx5ribCMqQKWXexahJZP0bVnDBjtORoCptihpyfx3wfcnsvibrJEteo1WtlCIjMj1BYeG6lkkFaGVq7di2eeeYZvPjiiwgGg5g5cyauueYalWNLS8xd6+3KkNky5Aazm8zxsWbLUCha1Eyl4iEbKO63VFMSoUezwlHWNc/jkUTITLCbTBWaxULqEzeZ4v1FkYKK5HZyTC05ol3rnZ/H6iZTc89NbG8yV6dTQkZAQwP8oZg5xZEytHfvXixduhRLly7Fnj17MG7cOPz617/GzJkzkZ+fn6gxphXmOWS/WIKKlKFoPzLnx5o/25zCnqg6Q04WAft73fr5iRpG9O2Cp+aejpN7uw8qVkHQlk3mV2QtpOrHkWRuMiNmKJpN5rWbTD9VJGaouTeZ8jpD3isgQRdri9fErQydf/75WLNmDYqLizF37lxcffXVGDhwYCLHlpa0FUCtP3erd9jdZE7QCzWGBVDfaK7nk6AAaifZZD7zoZMImqbhPBduLdVkWtxkfo4Z8ollSDKhIR5UFrzU0S1Dkf5kLmKGLEVu1Vlx6psiTWTdK1gaNA3RIHEfmIb0DXxKu8lyc3Px4osvYurUqQgq6LtDYtNWo1anfcTa+wxZZSEY0BAOCeOiVjEmM5ZFwMEY7ferZNydkMRzrCE6b0sK/WsZSqQS4oREVaAGgCvH9MWf3v8C5wxq2e9SFqNZq8Uy5Pw8AYWWIfP9Ud9EZmW4X0czApphafKD/hF0sdH2mriVoVdffTWR4yDNtHUBRmOG3JpXW36WEyLHCaMztHlsKpAtNscAahIPh2vrjcd6cLcfSWSsjhOsKdNqz12QnYFVN5+t9Jx66YXjjWFXMUMqLUNm9+JxRZYhIHovjj72FlVxrV7A1cJntOUm05UgtzfGqPbu7vhExQxJ1xliaj2Jg8O1DV4PIS7Ml79f3GR+WHDbw7AMNYUQDuvKkPNxJyKAGogoaYCali1+K3SYzJYhLhc+w5pa30qdIZcTzW0gtv75eoqo27pHdpQFUFMbIjE4/G1yKEP+qUDtrwW3PXRlyG3RRWsAtdvU+ujj4426ZUhtM18/6B/JHDPE1cJnaG24yYJ6TzG3u5Tmz5BVYHSlw6wMqcSSReMqgFrViEgqUV3X6PUQ4iLgEyVENobPK7JjpNZLteOw1BlSE6epjwtwn00G+CeuzBgD3WREFeYpZI8N0ie720nfpblbdBfJtNagTRlSGS8E2IrNuXCT0TJEYvHg9FMBAAunDPJ4JG1jns5eKiHWul+eDSNujNT6phD0ls1S7Tja6AbglFhuMreNWu3n9YOi6iZT2WtcVaAm6okngNpt/MDVZ56Ivt3ycP6QHlLH62PUY4ZU7wJkezLZL0A/pJoS/zFzZBnOHVyCbgX+TasH/OOeSmQF6kRgpNabAqhlMN9P3FpxzOdSlVpvP68f9I+0qDNEOoa2UuuNDvMub0idcjJx6Wl9pI/XJ7rfLUPJeEGSjsHvihDgzwrUybDjj9WOQ0Z+lmwyt73JTIcbqfUKblBRS4z6gpgy6BY0P7jsnMLlwmfEV3TR24nWMoBa7TSStwxZn6seFyEdieymQPk4JK9Hr4gGUJsbtTo/j/m7qiqQCJhT69W5ybxeE3RUeS+8gKuFz7C247BbOvwRnKZP9PpQgixDknWGzDccwJqaTEiyYb6svLRymj87Gda4aJ0hde04VMQf6ucz2nEoiBnSh+UXZSiZu9ZTGfIZWhsXYIZR
```

Comment:
I find the last sentence a bit confusing. It's good to mention that preprocess is not necessarily always the best approach, in fact I would write this  as a separate paragraph at the end of the notebook and expand it by pointing out that for a smaller dataset (or whatever the reason might be) using preprocess might be an overkill or even less efficient.